3 - Assignment/240326 - Rust Basic + Roguelike Game, Week 1/nested-arrays/src/lib.rs

```diff
@@ -1,5 +1,15 @@
 pub fn transpose(matrix: [[i32; 3]; 3]) -> [[i32; 3]; 3] {
-    todo!("Implement this")
+    let mut ret = [
+        [0,0,0],
+        [0,0,0],
+        [0,0,0],
+    ];
```

Comment:
더욱 간단한 방법으로 초기화할 수 있습니다.

```suggestion
    let mut ret = [[0; 3]; 3];
```